src/functions/durableEntity.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IEntityFunctionContext } from 'durable-functions/lib/src/classes';
```

Comment:
We should only be importing from the root of the package

Reply:
Yeah I agree. This is because those types are types that already existed on the durable package (not ones I defined in the new prog model), and the durable SDK is not great at exporting its types from the root of the package. There's no other way (currently) to import those types. Durable samples would just exclude those types from the samples altogether in the past

Reply:
> Durable samples would just exclude those types from the samples altogether in the past

I think this is the only option until the types are exposed from the root

Reply:
Updated so I only use types exported from the root package